### ISSUE TITLE
The B2 uses java.util.PropertyPermission which is going to be blocked in the future.

### ISSUE THREAD

**Issue by shaneargo** [BondUniversity/bond-ClassGroups#2]
We need to remove this from the bb-manifest:

````xml
<permission type="java.util.PropertyPermission" name="jboss.i18n.generate-proxies" actions="write" />
````

Off the top of my head, I do not know why it is currently required.

**Comment by shaneargo** [comment]
I think this almost certainly comes from hibernate. I have no desire to keep using hibernate, it was only used because this was developed prior to Bb's SimpleDAO stuff.

If someone wants to have a look at replacing it, feel free.

I guess it is possible that upgrading hibernate or changing a setting may also fix it.

**Comment by shaneargo** [comment]
Fixed by PR #5
